### DISCUSSION TITLE
Any plans on supporting backlinks?

### DISCUSSION THREAD

**Discussion by NightMachinery** [nvim-neorg/neorg#115] [Ideas]
Backlinks are the best idea of `org-roam`, IMHO. Any plans on supporting them as a first-class citizen?

**Comment by danymat** [comment]
That can be a great idea, that can be made in parallel of Zettelkasten note taking. 

I don't know however how backlinks should be used/displayed in order to be usefull. Do you have any ideas ?

**Reply by NightMachinery** [reply]
I personally use https://github.com/toshism/org-super-links ; Any header that has links to it, will have an ID in its PROPERTIES drawer (so no link rot), and backlinks are automatically added to the BACKLINKS drawer of the source heading.

**Reply by danymat** [reply]
> I personally use https://github.com/toshism/org-super-links ; Any header that has links to it, will have an ID in its PROPERTIES drawer (so no link rot), and backlinks are automatically added to the BACKLINKS drawer of the source heading. 
> 
> 
> 
> 

So if I understand correctly, a drawer is like a small view where you can do some actions too?

**Comment by NightMachinery** [comment]
[**@
        danymat
      **](https://github.com/danymat) commented on [Sep 19, 2021, 1:48 PM GMT+4:30](https://github.com/nvim-neorg/neorg/discussions/115#discussioncomment-1353211 "2021-09-19T09:18:25Z - Replied by Github Reply Comments"):
> > I personally use [toshism/org-super-links](https://github.com/toshism/org-super-links) ; Any header that has links to it, will have an ID in its PROPERTIES drawer (so no link rot), and backlinks are automatically added to the BACKLINKS drawer of the source heading.
> 
> So if I understand correctly, a drawer is like a small view where you can do some actions too?

No, a drawer is just a block that is usually hidden and not exported:
```org

* *links*
:PROPERTIES:
:ID:       b2912b13-562b-42ea-8706-0b38c0e1502c
:END:


** [[id:16fc5a7e-b439-4f84-9a53-bbd5112524d0][wow]]

* *wow*
:PROPERTIES:
:ID:       16fc5a7e-b439-4f84-9a53-bbd5112524d0
:END:
:BACKLINKS:
[[id:b2912b13-562b-42ea-8706-0b38c0e1502c][links]]
:END:
```
